packages/segmentedControl/components/SegmentedControlButton.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { cx } from "emotion";
+import shortid from "shortid";
+import { segmentedControlButton, segmentedControlButtonActive } from "../style";
+import { visuallyHidden, display } from "../../shared/styles/styleUtils";
+
+export interface SegmentedControlButtonProps {
+  /**
+   * Unique identifier used for the input button
+   */
+  id?: string;
+  /**
+   * Whether the button has been selected
+   */
+  isActive?: boolean;
+  /**
+   * The name passed to the input. Shared by all inputs in the segmented control
+   */
+  name?: string;
+  /**
+   * Callback for when a user makes a selection. Use the `onSelect` prop on the `SegmentedControl` instead, unless there is some extenuating circumstance.
+   */
+  onChange?: (e: React.SyntheticEvent<HTMLElement>) => void;
+  /**
+   * The value of the input
+   */
+  value: string;
+}
+
+const SegmentedControlButton: React.SFC<
+  SegmentedControlButtonProps
+> = props => {
+  const { id = shortid.generate(), isActive, onChange, name, value } = props;
+  return (
+    <label
+      className={cx(display("inline-block"), segmentedControlButton, {
+        [segmentedControlButtonActive]: props.isActive
+      })}
+      data-cy="segmentedControlButton"
+      htmlFor={id}
+    >
+      {/* TODO: more detail on why this is disabled */}
```

Comment:
care to explain now? or is this a todo for later?

Reply:
This is a TODO for now, I just forgot to do it 🙈 

There was a a11y linting error that was asking me to add ARIA props that would have been redundant and potentially create an awkward screenreader experience